helloworld.js

```diff
@@ -0,0 +1,23 @@
+var express = require('express')
+
+var app = express()
+
+app.get('/greet', sendGreeting)
+app.get('/sqrt', sendSqrt)
+
+var server = app.listen(3000, function () {
+	var host = server.address().address
+	var port = server.address().port
+
+	console.log('Example app listening at http://%s:%s', host, port)
+})
+
+function sendGreeting(request, response) {
+	response.send('hello world');
+}
```

Comment:
I really like these functions! Even if this is just an example, it would be great if we could refactor these into "routes," which we would import from other files. You can read this [tutorial](https://scotch.io/tutorials/learn-to-use-the-new-router-in-expressjs-4) to learn more about routes.